tox.ini

```diff
@@ -7,6 +7,7 @@ skip_missing_interpreters = true
 description = Run the tests
 deps =
     coverage[toml]
+    hypothesis
```

Comment:
What's your minimum version?

(I prefer to pin all my dependencies with `pip-compile`, but at least give it a minimum version)

Reply:
I have had trouble using `pip-compile` in the past because there's no way to cross compile. Either way, that would go in a `constraints.txt` file, this is for abstract dependencies.

I'm not really sure what my minimum version is here. I'm only using `hypothesis.strategies.sampled_from` and `hypothesis.strategies.given`, so I'm guessing it's a very early version of `hypothesis`. I'll just set the minimum to the most recent version, since I know that version works with Python 3.8.